sw.js

```diff
@@ -40,7 +40,7 @@ function updateCache(request, response) {
     return caches.open(CACHE).then(function (cache) {
         return cache.put(request, response);
     });
-}
+}  
```

Comment:
Is this change needed?

Reply:
Here also i have not added a code ,just added line here.


Thanks and Regards